data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/source/KafkaSource.java

```diff
@@ -241,7 +241,6 @@ private Properties getConsumerProperties(final TopicConsumerConfig topicConfig,
         }
         setConsumerTopicProperties(properties, topicConfig);
         setSchemaRegistryProperties(properties, topicConfig);
-        LOG.debug("Starting consumer with the properties : {}", properties);
```

Comment:
We can consider just having log without properties.

LOG.info("Starting kafka consumer");

Reply:
will SENSITIVE logger mask all the properties field ?

Reply:
Yes, see the unit tests:
https://github.com/opensearch-project/data-prepper/blob/e341d7d3c9a55d37de5f512d4e1572ba8a881b0a/data-prepper-core/src/test/java/org/opensearch/dataprepper/logging/SensitiveArgumentMaskingConverterTest.java